independent-projects/bootstrap/pom.xml

```diff
@@ -71,7 +71,7 @@
         <jboss-logmanager-embedded.version>1.0.11</jboss-logmanager-embedded.version>
         <slf4j-jboss-logmanager.version>1.1.0.Final</slf4j-jboss-logmanager.version>
         <slf4j-api.version>1.7.36</slf4j-api.version>
-        <graal-sdk.version>23.1.2</graal-sdk.version>
+        <graal-sdk.version>23.0.1</graal-sdk.version>
```

Comment:
The version here is being downgraded, is that intended?

Reply:
No the version was actually incorrect due to a bad search/replace by yours truly. So @galderz is fixing it in passing.

Reply:
That version was wrong. A search/replace gone wrong https://github.com/quarkusio/quarkus/pull/35055

Reply:
Right, thanks.